### ISSUE TITLE
Dealing with Dynamic Height

### ISSUE THREAD

**Issue by alexlafroscia** [yapplabs/ember-modal-dialog#83]
How would you go about keeping a modal centered when its height is determined by content that is loaded after rendering?  Is there any way to do that?

**Comment by zethussuen** [comment]
:+1:

**Comment by chrislopresto** [comment]
I would like to expose a nicer way to do this on a modal-by-modal basis. In the meantime, you can do something along these lines, as Hubspot Tether provides a global position method.

```
actions: {
    somethingHappenedThatMeritsModalRepositioning() {
      Ember.run.schedule('afterRender', function() {
        Tether.position();
      });
    }
  }
```

**Comment by mangatinanda** [comment]
:+1: Any other ways to implement this without using `tether-dialog` component? I need this feature badly..  .

**Comment by primoz-k** [comment]
This is still not fixed, right?

**Comment by lukemelia** [comment]
@chrislopresto's suggestion remains the best known approach AFAIK

**Comment by lukemelia** [comment]
Fixed by an alternative CSS approach in [2.2.0](https://github.com/yapplabs/ember-modal-dialog/releases/tag/v2.2.0)
